ibis/omniscidb/ddl.py

```diff
@@ -340,6 +341,53 @@ def compile(self):
         return self._wrap_command(action)
 
 
+class AddColumn(AlterTable):
+    """Add Column class."""
+
+    def __init__(self, table_name, column_name, data_type):
+        super().__init__(table_name)
+        self.column_name = column_name
+        self.data_type = data_type
+
+    def compile(self):
+        """Compile the Add Column expression.
+
+        Returns
+        -------
+        string
+        """
+        unsupported_types = ['POINT',
```

Comment:
Why these types are unsupported? Omnisci supports geo functions and types, there are tests for them.

Reply:
I read it at the [Omnisci documentation](https://docs.omnisci.com/latest/5_tables.html)

Reply:
I found it in examples from doc page
`ALTER TABLE t ADD COLUMN pt_dropoff POINT DEFAULT 'point(0 0)';`

Reply:
Clause 'Note' say that "Currently, OmniSci does not support adding a geo column type (POINT, LINESTRING, POLYGON, or MULTIPOLYGON) to a table"

Reply:
Sorry, didn't notice

Reply:
I checked it, it is working fine on omnisci master branch.

Reply:
Then may delete it, thanks